### DISCUSSION TITLE
Intregration of scATAC datasets when fragment file is missing

### DISCUSSION THREAD

**Discussion by katimbach** [stuart-lab/signac#1181] [Q&A] [answered]
Hi everybody, 

I want a bit of advice about a problem I've run into in my analysis. I am trying to integrate my scATAC dataset (currently Harmony integration of 4 samples) with a similar scATAC-seq dataset from a repository for labeling and comparison purposes. However, I am having trouble calculating consensus peaks between my dataset and the one downloaded from GEO, as the GEO data did not have all of the 10X output files, and lacks the fragment.tsv.gz file. Instead, it seems to just have the contents from filtered_peak_bc_matrix and some cell metadata. 

So far, the only documentation I have seen regarding calculating consensus peaks in two datasets relies on the Fragment Object, which I cannot generate without the fragment.tsv.gz. Is there a workaround here to make a pseudo fragment file just using the peak/cell matrix? Or is this approach just not feasible with the format of the repository data I currently have? 

Thanks in advance for the help!

**Comment by timoast** [comment, marked as answer]
The fragment file is required to quantify counts in genomic regions. To perform integration, you need to have the same features quantified across all the datasets to be integrated. If you don't have the fragment file for the dataset from GEO, you could instead quantify the GEO dataset's peaks in your samples, for which I'm assuming you do have fragment files.
